### PR TITLE
[INFRA/IIGO] Removing Globals : Part 1 of 4

### PR DESCRIPTION
**Pull request by preetl**
# Summary

Maps in IIGO moved to game state. Lots more to come!

## Additional Information

Related to issue #343 and #310 

## Test Plan

Tests still pass.

### CODE REVIEW

File: internal/server/iigointernal/orchestration.go [hunk 1]
```diff
@@ -9,35 +9,30 @@ import (
...
-// iigoClients holds pointers to all the clients
-var iigoClients map[shared.ClientID]baseclient.Client
-
 //monitoring holds the monitoring object that is used across turns
 var monitoring monitor
 
 // RunIIGO runs all iigo function in sequence
 func RunIIGO(logger shared.Logger, g *gamestate.GameState, clientMap *map[shared.ClientID]baseclient.Client, gameConf *config.Config) (IIGOSuccessful bool, StatusDescription string) {
 
+	iIGOClients := *clientMap
```
Comment (lhl2617) [inline internal/server/iigointernal/orchestration.go:18]: very strange casing

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (lhl2617) [inline yarn.lock]: delete. someone ran `yarn start` in root ;)

File: node_modules/.yarn-integrity [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (lhl2617) [inline node_modules/.yarn-integrity]: delete

File: yarn.lock (hunk 2 above)
Comment (preetl) [reply yarn.lock]: whoopsie

**Review by lhl2617 — APPROVED** [review]
lgtm

image

**Pull request merged**